analyzers/tests/SonarAnalyzer.UnitTest/Rules/NativeMethodsShouldBeWrappedTest.cs

```diff
@@ -47,6 +47,7 @@ public class NativeMethodsShouldBeWrappedTest
                 .AddPaths("NativeMethodsShouldBeWrapped.CSharp11.cs")
                 .AddPaths("NativeMethodsShouldBeWrapped.CSharp11.SourceGenerator.cs")
                 .WithOptions(ParseOptionsHelper.FromCSharp11)
+                .WithConcurrentAnalysis(false)
```

Comment:
[education] why is this necessary?

Reply:
The `<autogenerated />` comment gets wrapped in the concurrent namespace and the logic to detect autogenerated code is no longer working. https://github.com/SonarSource/sonar-dotnet/pull/6211 should fix this.